pkg/cnb/project_descriptor_test.go

```diff
@@ -207,5 +221,223 @@ uri = "check-this-out.com"
 			})
 		})
 
+		when("the descriptor has groups", func() {
+			it.Before(func() {
+				ioutil.WriteFile(projectToml, []byte(`
+[[build.groups]]
+id = "cool-buildpack"
+version = "v4.2"
+uri = "check-this-out.com"
+				`), 0644)
+			})
+			it("it ignores the table", func() {
+				assert.Nil(t, cnb.ProcessProjectDescriptor(appDir, platformDir, logger))
```

Comment:
should we log that the key is being ignored similar to buildpacks and builders?

Reply:
I have purposefully not logged it since it's an artifact of our implementation where I have both buildpacks and groups in the same struct and they are used in one version but not the other.

For builders and others we purposefully ignore a valid specced project descriptor because we don't support certain functionality.